packages/react-core/src/demos/Toolbar.md

```diff
@@ -0,0 +1,585 @@
+                Options
+              </DropdownToggle>
+            }
+            isOpen={optionExpanded}
+            dropdownItems={optionDropdownItems}
+          />
+        </ToolbarItem>
+        <ToolbarItem visibility={{default: 'hidden', '2xl': 'visible'}}>
+          <Button variant={isPaused? "plain": "link"} onClick={this.pauseOrStart}>
+            {isPaused? <PlayIcon /> : <PauseIcon />}
+            {` Log Stream`}
+          </Button>
+        </ToolbarItem>
+      </React.Fragment>
+    );
+
+    const leftAlignedItemsMobile = (
+      <React.Fragment>
+        <ToolbarItem visibility={{default: 'visible', '2xl': 'hidden'}}>
+          <Tooltip
+            position="top"
+            content={<div>Select Pod</div>}
+          >
+            <Select
+              onToggle={this.onContainerToggleMobile}
+              onSelect={this.onContainerSelectMobile}
+              selections={containerSelected}
+              isOpen={containerExpandedMobile}
+              customContent={selectDropdownContent}
+              placeholderText={selectToggleContent({ showText: false })}
+            />
+          </Tooltip>
+        </ToolbarItem>
+        <ToolbarItem visibility={{default: 'visible', '2xl': 'hidden'}}>
+          <Tooltip
+            position="top"
+            content={<div>Options</div>}
+          >
+            <Dropdown
+              toggle={
+                <DropdownToggle id='option-toggle-mobile' onToggle={this.onOptionToggleMobile} icon={<CogIcon />} />
+              }
+              isOpen={optionExpandedMobile}
+              dropdownItems={optionDropdownItems}
+            />
+          </Tooltip>
+        </ToolbarItem>
+        <ToolbarItem visibility={{default: 'visible', '2xl': 'hidden'}}>
+          <Tooltip
+            position="top"
+            content={<div>{isPaused? "Start log" : "Pause log"}</div>}
+          >
+            <Button variant="plain" onClick={this.pauseOrStart}>
+              {isPaused? <PlayIcon /> : <PauseIcon />}
+            </Button>
+          </Tooltip>
+        </ToolbarItem>
+      </React.Fragment>
+    );
+
+    const leftAlignedItems = (
+      <React.Fragment>
+        {leftAlignedItemsDesktop}
+        {leftAlignedItemsMobile}
+      </React.Fragment>
+    );
+
+    const rightAlignedItemsDesktop = (
+      <React.Fragment>
+        <ToolbarItem visibility={{default: 'hidden', lg: 'visible'}}>
+          {LogsSearchInput}
+        </ToolbarItem>
+        <ToolbarItem visibility={{default: 'hidden', '2xl': 'visible'}}>
+          <Dropdown
+            onSelect={this.onExternalSelect}
+            toggle={
+              <DropdownToggle
+                splitButtonItems={[
+                  <DropdownToggleAction key="action-7" onClick={this.onExternalClick}>
+                    External Logs
+                  </DropdownToggleAction>
+                ]}
+                splitButtonVariant="action"
```

Comment:
Non-split, basic dropdowns may work a bit better for the External Logs and Download dropdowns here. I see now that the mockup had a divider line which was deceiving, but I can't think of a scenario (at least in OpenShift) where we'd need these to be split.